v4/source/webhooks.yaml

```
@@ -19,6 +21,9 @@
                   type: string
                   description: The ID of a public channel or private group that receives
                     the webhook payloads.
+                user_id:
+                  type: string
+                  description: The ID of the owner of the webhook if different than the requester. Required in [local mode](https://docs.mattermost.com/administration/mmctl-cli-tool.html#local-mode).
```

nit: not an language expert but this sounded better to me 😅 
```suggestion
                  description: The ID of the owner of the webhook if different than the requester. Required for [local mode](https://docs.mattermost.com/administration/mmctl-cli-tool.html#local-mode).
```